src/current/v24.2/troubleshoot-lock-contention.md

```diff
@@ -0,0 +1,395 @@
+                                 | waiting                      | `UPDATE t SET v=2032 WHERE k=2;`
+                                 |                              | waiting
+`COMMIT;`                        | unblocked to proceed         | unblocked to proceed
+success, k=2,v=2012              | k=2, v=2012                  |
+                                 | `COMMIT;`                    | `COMMIT;`
+                                 | success                      | success                                 
+                                 |                              | `SELECT * FROM t WHERE k=2;`
+                                 |                              | k=2, v=2032
+
+### SQL statements
+
+To reproduce Example 1 in CockroachDB in preparation for the next section on how to [identify waiting and blocking transactions](#identify-waiting-and-blocking-transactions) using the Insights page of the Cloud Console or the DB Console, execute the following SQL statements in the given order in the specified terminal.
+
+**Terminal 1**
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+BEGIN;
+UPDATE t SET v=2012 WHERE k=2; -- lock k=2
+~~~
+
+**Terminal 2**
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+BEGIN;
+SELECT * FROM t WHERE k=2; -- waiting read
+~~~
+
+**Terminal 3**
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+BEGIN;
+UPDATE t SET v=2032 WHERE k=2; -- waiting write
+~~~
+
+**Terminal 1**
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+COMMIT;
+~~~
+
+**Terminal 2**
+
+When Transaction 1, releases key `k=2`, Transaction 2 should output the following:
+
+~~~
+  k |  v
+----+-------
+  2 | 2012
+(1 row)
+~~~
+
+`COMMIT` Transaction 2:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+COMMIT;
+~~~
+
+**Terminal 3**
+
+`COMMIT` Transaction 3 and verify that the `UPDATE` has succeeded:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+COMMIT;
+SELECT * FROM t where k=2;
+~~~
+
+The `SELECT` statement should output the following:
+
+~~~
+  k |  v
+----+-------
+  2 | 2032
+(1 row)
+~~~
+
+## Identify waiting and blocking transactions
+
+This section of the tutorial uses the [**Insights** page]({% link {{ page.version.version }}/ui-insights-page.md %}#transaction-executions-view) of the DB Console to identify waiting and block transactions in the demo cluster. With a CockroachDB Cloud cluster, the Cloud Console has a similar [**Insights** page]({% link cockroachcloud/insights-page.md %}#transaction-executions-view). You can also use the [`crdb_internal`]({% link {{ page.version.version }}/performance-recipes.md %}#identify-transactions-and-objects-that-experienced-lock-contention) system catalog to view tables and indexes that experienced contention.
+
+While this tutorial uses the data from [Example 1](#example-1), when troubleshooting lock contention in your own workload, you can adapt the following steps using the DB Console or the Cloud Console.
+
+### High Contention insight events
```

Comment:
Nit: I don't think we call out insights as 'events' at all in the docs or UI. Could we just say insights? Just to avoid overloading the meaning of "Events"

Note we do have "Insight Type" in the UI if we need to reference